test/connection.test.ts

```diff
@@ -520,6 +528,23 @@ describe('Connection', function () {
       assert.calledOnce(spy)
       assert.calledWith(spy, '58')
     })
+
+    it('should get a receipt for each fulfilled packet', async function () {
+      const clientStream = this.clientConn.createStream()
+      const spy = sinon.spy(clientStream, '_setReceipt')
+      await clientStream.sendTotal(1002)
```

Comment:
should this also check that the receipt is exposed from the client stream?

Reply:
added check
https://github.com/interledgerjs/ilp-protocol-stream/pull/133/commits/66e008e12f951cc1a0ec2bd04f293f6a0b027a41